extensions/apps/extensions/src/components/pages/info-page/sub-pages/releases-page.tsx

```diff
@@ -56,17 +56,16 @@ export const ReleasesPage = (props: ReleasesPageProps) => {
 
   const handleLoadMoreReleases = () => {
     if (releasesReq.data?.akashaAppReleaseIndex?.edges.length < releasesCount) {
-      return;
+      releasesReq.fetchMore({
+        variables: {
+          after: releasesReq.data?.akashaAppReleaseIndex?.pageInfo.endCursor,
```

Comment:
this will error if after is null

Reply:
hmm, but can it be null? because the if condition already guards against empty response

Reply:
I just reviewed the block here